robotoff/slack.py

```diff
@@ -244,6 +238,24 @@ def _post_message(
             )
 
 
+class NoopSlackNotifier(SlackNotifier):
+    """NoopSlackNotifier is a NOOP SlackNotifier used in dev/local executions of Robotoff."""
+
+    def __init__(self):
+        pass
```

Comment:
To avoid deepsource error, maybe use:

```suggestion
    def __init__(self):
        super().__init__("")
```

(but really, I think deepsource is really tedious here)